recipes-framework/libedgetpu/libedgetpu_16.0.bb

```diff
@@ -27,3 +27,4 @@ SRC_URI = "https://github.com/google-coral/${PN}/archive/refs/tags/release-${REL
 SRC_URI[lib.sha256sum] = "86a6e654e093c204b4fb579a60773bfa080f095c9cbb3a2c114ca4a13e0b15eb"
 
 PROVIDES += "libedgetpu1-std libedgetpu1-max libedgetpu-dev"
+RPROVIDES += "${PROVIDES}"
```

Comment:
Weren't we discussing to make this a default in Isar? Looks like we now need to replicate this pattern all over the place.

Reply:
Yes, IIRC that is not yet on the list, but we are at least internally testing this. I added it here as it really makes a difference with the bazel based components and also want to decouple this somehow from the ISAR release. I vote to keep it here for now. Once we have a new ISAR release with that included, we can revert the patch.

Reply:
I dropped the commit from this series until further clarification with upstream ISAR